hedera-node/src/main/java/com/hedera/services/store/contracts/HederaStackedWorldStateUpdater.java

```diff
@@ -70,7 +70,7 @@ public HederaStackedWorldStateUpdater(
 		this.dynamicProperties = dynamicProperties;
 	}
 
-	public boolean hasMutableLedgers() {
+	public boolean isInTransaction() {
```

Comment:
Q: Is the renaming a result of inference?
That is we infer if it's mutable then it's `inTransaction`?
The previous name seemed more appropriate of the check but i guess the renaming is to call out the result?

Reply:
In a discussion , we thought query can't change state and `mutability` meant that it is in a `Transaction`. So thought of renaming the function call to be more explicit.